src/toil/batchSystems/mesos/batchSystem.py

```diff
@@ -21,6 +21,7 @@
 import pwd
 import socket
 import time
+from bisect import bisect
 from collections import defaultdict
 from operator import attrgetter
```

Comment:
Additional nitpick: these three imports (bisect, defaultdict, attrgetter) aren't used anywhere in this file according to pyflakes. I think the part that used defaultdict was removed in this PR, and I assume the others are from a first draft of the new code.